cbw_api_toolbox/cbw_api.py

```diff
@@ -167,3 +169,42 @@ def update_remote_access(self, remote_access_id, info):
 
         logging.error("Error update remote access")
         return False
+
+    def import_remote_accesses_xlsx(self, file_xlsx):
```

Comment:
I don't understand why this code is here. I don't think that the purspose of this code is "Class used to communicate with the CBW API".

You need to add a new class that read the xlsx file and use the method CbwApi.create_remote_access